CMakeLists.txt.in

```diff
@@ -0,0 +1,215 @@
+cmake_minimum_required(VERSION 3.5)
+project(build-deps NONE)
+
+cmake_host_system_information(
+  RESULT build_concurrency_factor
+  QUERY NUMBER_OF_LOGICAL_CORES)
+set(make_command make -j ${build_concurrency_factor})
+
+include(ExternalProject)
+
+ExternalProject_Add(boost_ext
+  # The 1.67.0 release has a bug in Boost Lockfree around a missing header.
+  URL https://dl.bintray.com/boostorg/release/1.64.0/source/boost_1_64_0.tar.gz
+  URL_MD5 319c6ffbbeccc366f14bb68767a6db79
+  PATCH_COMMAND
+    ./bootstrap.sh
+    --prefix=@SMF_DEPS_INSTALL_DIR@
+    --with-libraries=atomic,chrono,date_time,filesystem,program_options,system,test,thread
+  CONFIGURE_COMMAND ""
+  BUILD_COMMAND ""
+  INSTALL_COMMAND
+    ${CMAKE_COMMAND} -E chdir <SOURCE_DIR>
+    ./b2
+    -j ${build_concurrency_factor}
+    --layout=system
+    --build-dir=<BINARY_DIR>
+    install
+    variant=debug
+    link=shared
```

Comment:
change to static and variant=release for boost. I fixed this in our deps which differ slightly from seastar - we use `as static as possible`

Reply:
by that i mean `link=static` `variant=release`